pkg/cmd/run.go

```diff
@@ -50,6 +50,8 @@ type fallbackOptions struct {
 	passphrase         string
 }
 
+var secretsToInclude []string
```

Comment:
Commit message: in a changelog, it's not super clear what "Add support for --include flag" is referring to. Something like "Allow `run` command to inject subset of secrets" is still short but provides a lot more context for the reader.